### ISSUE TITLE
Node6 does not support syntax of trailing comma

### ISSUE THREAD

**Issue by chenhebing** [SimenB/add-asset-html-webpack-plugin#147]
Hi, when use this plugin, I get the following error.
```
/node_modules/add-asset-html-webpack-plugin/lib/index.js:121
      );
      ^

SyntaxError: Unexpected token )
```
My node version is v6.16.0, and the engines node of this plugin is greater or equal to 6. But node6 does not support trailing comma syntax.

I upgrade node to v8.x.x, this plugin work well.

**Comment by SimenB** [comment]
Gah, I'm dumb. I run prettier on the output from babel, which adds the trailing commas... Will fix, sorry!

**Comment by chenhebing** [comment]
> 🎉 This issue has been resolved in version 3.1.3 🎉
> 
> The release is available on:
> 
> * [npm package (@latest dist-tag)](https://www.npmjs.com/package/add-asset-html-webpack-plugin)
> * [GitHub release](https://github.com/SimenB/add-asset-html-webpack-plugin/releases/tag/v3.1.3)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀


> Gah, I'm dumb. I run prettier on the output from babel, which adds the trailing commas... Will fix, sorry!

Thanks  so much!
